pools/usecase/pools_usecase.go

```diff
@@ -312,7 +312,7 @@ func (p *poolsUseCase) StorePools(pools []sqsdomain.PoolI) error {
 		// If orderbook, update top liquidity pool for base and quote denom if it has higher liquidity capitalization.
 		sqsModel := pool.GetSQSPoolModel()
 		cosmWasmPoolModel := sqsModel.CosmWasmPoolModel
-		if cosmWasmPoolModel != nil && cosmWasmPoolModel.IsOrderbook() {
+		if cosmWasmPoolModel != nil && cosmWasmPoolModel.Data.Orderbook != nil && cosmWasmPoolModel.IsOrderbook() {
```

Comment:
Oh wow - did this have a bug?

Reply:
Initially I thought it's because of outdated `osmosisd` local version, but I am getting same on `dev env` too. I probably would need to check this behaviour against `v25.x` branch, it's on my list.

Reply:
I see - nice catch - let's separate it into its own PR for quick merge and deployment 🙏

Reply:
Created a separate [PR-411](https://github.com/osmosis-labs/sqs/pull/411) :rocket: